### DISCUSSION TITLE
Implement Chafa?

### DISCUSSION THREAD

**Discussion by madmedicnl** [gokcehan/lf#1801] [Q&A] [answered]
Would it be possible to add an option to use chafa as an image previewer?
[https://github.com/hpjansson/chafa/](https://github.com/hpjansson/chafa/)
Chafa also works in powershell and that would really be a bliss to have.

**Comment by valoq** [comment, marked as answer]
Image previews can already be configured through the lfrc file.
The wiki describes how to use chafa in lf:
https://github.com/gokcehan/lf/wiki/Previews#with-sixel
